docs/src/references/c8y-configuration-management.md

```diff
@@ -162,4 +162,233 @@ Note that:
   for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
 * Since the type of configuration file is used as an MQTT topic name, the characters `#` and `+` cannot be used in a type name.
   If such a character is used in a type name (or in the path of a configuration file without explicit type),
-  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
\ No newline at end of file
+  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
+* Notifications for child-devices and their external devices are handled differently, as configuration files are fetched/pushed from/to external devices via HTTP file transfer. Thereby notifications are part of the file transfer contract (details see section [Fetching/Pushing configuration file](#fetchingpushing-configuration-file-fromto-the-external-device) below).
```

Comment:
> Notifications for child-devices and their external devices are handled differently

I can understand that some differences are required but why so many? Why no more paths to configuration files? Why only *opaque* configuration types?

Reply:
I am going to replace the `type` with the `path` as unique reference between the child-device and the plugin.

Then the behaviour for child-devices and the thin-edge device is more similar:
In both cases each configuration file has a `path`, but the difference is for a child-device that the path is treated as path pointing to the child-devices filesystem. And it's finally on the child-device whether it understand that `path` as a filesystem path, or something different (e.g. some database key, a variable name, ...).

Reply:
> I am going to replace the `type` with the `path` as unique reference between the child-device and the plugin.

That change is applied now.